apps/judicial-system/web/src/components/IndictmentCaseFilesList/IndictmentCaseFilesList.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useContext } from 'react'
+import { useIntl } from 'react-intl'
+
+import {
+  Case,
+  CaseFile,
+  CaseFileCategory,
+  completedCaseStates,
+  isCourtRole,
+} from '@island.is/judicial-system/types'
+import { Box, Text } from '@island.is/island-ui/core'
+
+import PdfButton from '../PdfButton/PdfButton'
+import { caseFiles } from '../../routes/Prosecutor/Indictments/CaseFiles/CaseFiles.strings'
+import { indictmentCaseFilesList as strings } from './IndictmentCaseFilesList.strings'
+import { useFileList } from '../../utils/hooks'
+import { UserContext } from '../UserProvider/UserProvider'
+import SectionHeading from '../SectionHeading/SectionHeading'
+import * as styles from './IndictmentCaseFilesList.css'
+import { courtRecord } from '../../routes/Court/Indictments/CourtRecord/CourtRecord.strings'
```

Comment:
Consider moving all strings to `indictmentCaseFilesList.strings.ts`

Reply:
These strings are being used on the "Þingbók og dómur" screen as well. Moving them means duplicating them.